go.mod

```diff
@@ -57,15 +58,14 @@ require (
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/sirupsen/logrus v1.9.0 // indirect
 	github.com/stretchr/testify v1.8.2 // indirect
-	golang.org/x/crypto v0.9.0 // indirect
+	golang.org/x/crypto v0.14.0 // indirect
 	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20230530153820-e85fd2cbaebc // indirect
 	google.golang.org/genproto/googleapis/api v0.0.0-20230530153820-e85fd2cbaebc // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20230530153820-e85fd2cbaebc // indirect
-	google.golang.org/grpc v1.55.0 // indirect
+	google.golang.org/grpc v1.56.3 // indirect
```

Comment:
We should not directly upgrade these versions.

```bash
go mod why google.golang.org/grpc
# google.golang.org/grpc
github.com/googlecloudplatform/gcsfuse/internal/fs/wrappers
cloud.google.com/go/storage
google.golang.org/grpc
```

Ideally, these versions should be upgraded by go-storage-client library and then we will upgrade the storage client library.

Reply:
The one which they are using for grpc is 1.59.0 - https://github.com/googleapis/google-cloud-go/blob/main/storage/go.mod#L20C2-L20C32

Reply:
Yeah, we should consider upgrading go-storage client instead of upgrading individual dependency of go-storage-client.

Reply:
Yes, upgraded go storage client.